lab-gurpreet/lib/server.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
```

Comment:
Generally, it's best practice to have your server file in your project root, not in your 'lib' directory.